mods/binoculars/init.lua

```diff
@@ -51,16 +50,16 @@ local function cyclic_update()
 	for _, player in ipairs(minetest.get_connected_players()) do
 		binoculars.update_player_property(player)
 	end
-	minetest.after(.5, cyclic_update)
+	minetest.after(4.7, cyclic_update)
 end
 
-minetest.after(.5, cyclic_update)
+minetest.after(4.7, cyclic_update)
 
 
 -- Binoculars item
 
 minetest.register_craftitem("binoculars:binoculars", {
-	description = "Binoculars\nUse with 'Zoom' key",
+	description = S("Binoculars") .. "\n" .. S("Use with 'Zoom' key"),
 	inventory_image = "binoculars_binoculars.png",
 	stack_max = 1,
 
```

Comment:
All this really does is undo https://github.com/Elkien3/citysim_game/commit/023f987756752056844c97cd74c8715bc4bdb077

Reply:
fixed.